addons/vectors/test_vectors.sqf

```diff
@@ -12,13 +12,28 @@ private _fnc_vectorEquals = {
     if ((count _vector1) != (count _vector2)) exitWith {false};
     private _equal = true;
     {
-        if ((abs (_x - (_vector2 select _forEachIndex))) > 0.00001) then {
```

Comment:
can be replaced by findIf I think.

Reply:
`findif` does not support `_forEachIndex`

Reply:
No need to, findIf is kind of hard to read.